### DISCUSSION TITLE
Support for determining winners of Supplementary Vote elections

### DISCUSSION THREAD

**Discussion by russss** [DemocracyClub/yournextrepresentative#1773] [Ideas]
This is probably the next most significant voting system after FPTP, given the increasing popularity of elected mayors.

We need to record the number of first and second preference votes.

**Comment by symroe** [comment]
@pmk01 wrote some of this up in a doc here: https://docs.google.com/document/d/1qeWUytcOY4kWv2LokwYWE-pL7m7_3znCnwb9RIr45Zc/edit?usp=sharing

**Comment by VirginiaDooley** [comment]
@pmk01 is this a priority for May 2022?

**Comment by pmk01** [comment]
No, but the opposite will be. Unless there's a major U-Turn it looks like the supplementary vote will be abolished by 2022, it's in the elections bill currently going through parliament.

**Comment by VirginiaDooley** [comment]
Thanks. I should have asked: Is this still relevant, even for after SOPN day? 
We can create an issue for SV once we know the decision from parliament.

**Comment by pmk01** [comment]
No, this issue is probably irrelevant now, unless we want to go back to add results to past elections. The elections bill should be passed soon, so I think we should hold on this for a couple of weeks and then come back when we know what's what.
